gadget/install/content.go

```diff
@@ -110,5 +111,22 @@ func writeFilesystemContent(laidOut *gadget.LaidOutStructure, fsDevice string, o
 		return fmt.Errorf("cannot create filesystem image: %v", err)
 	}
 
+	// For data partition, copy drivers tree if it has been created, so
+	// kernel drivers are available on first boot of installed system.
+	if laidOut.Role() == gadget.SystemData {
+		src := dirs.SnapKernelDriversTreesDirUnder(dirs.GlobalRootDir)
```

Comment:
what's the source of this? the ephemeral system? what happens in the case of installation from snap-bootstrap?

Reply:
I am currently thinking that this is problematic for cases when we install using the snapd API, and will change the approach, possible by calling `kernel.EnsureKernelDriversTree()` from here.